mat/cholesky.go

```diff
@@ -390,10 +437,10 @@ func (c *Cholesky) Scale(f float64, orig *Cholesky) {
 // that k > w' A^-1 w. If this condition does not hold then ExtendVecSym will
 // return false and the receiver will not be updated.
 //
-// ExtendVecSym will panic if v.Len() != a.Size()+1 or if a does not contain
+// ExtendVecSym will panic if v.Len() != a.Symmetric()+1 or if a does not contain
 // a valid decomposition.
 func (chol *Cholesky) ExtendVecSym(a *Cholesky, v Vector) (ok bool) {
-	n := a.Size()
+	n := a.Symmetric()
```

Comment:
You are going to get a conflict here.

Reply:
Thanks. We can submit the other ones first and then I'll pull and fix.